### PR TITLE
Add configuration file for `pep8_speaks`

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

**Comment by soupault** [comment]
From CI:
```
Partial import of skimage during the build process.
Missing from source distribution:
   ./.pep8speaks.yml
Please update MANIFEST.in

The command "tools/check_sdist.py" failed and exited with 1 during .
```

File: .pep8speaks.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+scanner:
+  diff_only: True  # Errors caused by only the patch are shown, not the whole file
+
+pycodestyle:
+  max-line-length: 80  # Default is 79 in PEP8
```
Comment (jni) [inline .pep8speaks.yml]: Why 80 and not 79?

Comment (stefanv) [inline .pep8speaks.yml]: Some editors wrap onto exactly 80, so it's a bit annoying to be nitpicky about this.  I feel pretty ambivalent about 79 or 80—it must just display properly on the terminal.

Comment (stefanv) [inline .pep8speaks.yml]: I made this 79.

**Comment by stefanv** [comment]
Updated.

File: .pep8speaks.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+scanner:
+  diff_only: True  # Errors caused by only the patch are shown, not the whole file
+
+pycodestyle:
+  ignore:  # Errors and warnings to ignore
+    - W391
```
Comment (sciunto) [inline .pep8speaks.yml]: Could you add a comment to specify what the code means?

**Comment by sciunto** [comment]
@soupault I made the changed I wanted. You can press the green button.

**Pull request merged**